src/controllers/urls.ts

```diff
@@ -72,11 +84,22 @@ export const findUrlByCodeInt = async (codeInt: number) =>
     },
   })
 
-export const getAllUrlsForUser = async (user: UserAttributes) => {
-  const urls = await URLs.findAll({
+export const getAllUrlsForUser = async (
+  user: UserAttributes, 
+  limit: LimitingOptions
```

Comment:
this param should be called `page` and not `limit` as offset and **limit** are two parts of page options